### DISCUSSION TITLE
Mikado and web components

### DISCUSSION THREAD

**Discussion by GHNewbiee** [nextapps-de/mikado#69] [Ideas]
Hi, 

- @ts-thomas, what are your thoughts about combining `mikado` lib with *web components*? There is not a single reference in doc about `Shadow DOM`.
- has someone tried using `mikado` in `Shadow DOM` or any basic wc lib?

Looking forward to reading thoughts and experiences. *Tia*

**Comment by ts-thomas** [comment]
Shadow DOM is now supported by passing the option `shadow: true`, this feature will be extended soon to define style and script tags on the shadow root level.
